Header header
sensor_msgs/Image image_left
sensor_msgs/Image image_right
geometry_msgs/Pose pose
bool tracking_success